packages/cli/src/config/auth.test.ts

```
@@ -0,0 +1,438 @@
+    it('should handle null and undefined values', () => {
+      expect(isValidApiKey(null as any)).toBe(false);
+      expect(isValidApiKey(undefined as any)).toBe(false);
+    });
+
+    it('should handle whitespace in API keys', () => {
+      expect(isValidApiKey(' AIza1234567890abcdefghijklmnopqrstuvwxyz ')).toBe(false);
+      expect(isValidApiKey('AIza1234567890abcdefghijklmno pqrstuvwxyz')).toBe(false);
+    });
+
+    it('should validate API key length requirements', () => {
+      const validKey = 'AIza' + 'a'.repeat(35); // 39 chars total
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)